docs/deprecations.md

```diff
@@ -26,6 +26,7 @@ being deprecated.
 | [ClusterTask is deprecated](https://github.com/tektoncd/pipeline/issues/4476)                                                                                                                                                | v0.41.0                                                              | Beta                                                                                                   | July 13, 2023                       |
 | [`pipelineRef.bundle` and `taskRef.bundle` are deprecated](https://github.com/tektoncd/pipeline/issues/5514)                                                                                                                 | v0.41.0                                                              | Alpha                                                                                                  | July 13, 2023                       |
 | [The `config-trusted-resources` configMap is deprecated](https://github.com/tektoncd/pipeline/issues/5852)                                                                                                                 | v0.45.0                                                              | Alpha                                                                                                  | v0.46.0                       |
+| [TaskRun.Status.ResourcesResult is deprecated and tombstoned](https://github.com/tektoncd/pipeline/issues/6325)                                                                                                                 | v0.46.0                                                              | Alpha                                                                                                  | TODO: TBD                       |
```

Comment:
I would add this under "deprecated PipelineResources related features" instead of here

Reply:
Thanks Lee. Moved the the section below.